main/guides/js-programming/hardened-js.md

```diff
@@ -94,20 +94,47 @@ recommended style for writing JavaScript smart contracts.
 This `eslint` configuration provides tool support.
 :::
 
-1. If not already configured, run `yarn add eslint @jessie.js/eslint-plugin`
-2. If not already configured, add the following to your `package.json`:
+1. If working from an empty directory, a package.json file must first be created by running `yarn init` or `yarn init -y`.
+2. From there, we can install eslint into our project along with the jessie.js eslint-plugin by running `yarn add eslint @jessie.js/eslint-plugin`.
+3. The final step is to set up our project's eslint configuration inside of the package.json file by adding in the following code block.
 
 ```json
+"eslintConfig" : {
+  "parserOptions": {
+      "sourceType": "module",
+      "ecmaVersion": 6
+  },
+  "extends": [
+    "plugin:@jessie.js/recommended"
+  ]
+}
+```
+
+Now, the contents of the package.json file should look similiar to the snippet below.
+
+```json
+{
+  "name": "eslint-config-test",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "type": "module",
+  "devDependencies": {
+    "@jessie.js/eslint-plugin": "^0.1.3",
+    "eslint": "^8.6.0"
+  },
   "eslintConfig": {
-    "extends": [
-      "@jessie.js"
-    ]
+    "parserOptions": { "sourceType": "module", "ecmaVersion": 6 },
+    "extends": ["plugin:@jessie.js/recommended"]
   }
+}
```

Comment:
This may seem like overkill but IMO seeing the entire contents of the file best ensures the reader avoids any issues.